src/centralserver/admin-ui/src/views/Settings/Settings.vue

```diff
@@ -25,9 +25,9 @@
  -->
 <template>
   <div class="xrd-sub-view-wrapper">
-    <div class="content xrd-view-common">
+    <v-container class="xrd-view-common mt-7">
       <router-view />
-    </div>
+    </v-container>
   </div>
```

Comment:
It probably could, yeah. I'm not sure if it is needed now that the same can be achieved with the internal `v-container` component with `mt-7` class 🤷‍♂️

I do think that it would be more explicit to use a well named component, for example `MainContentArea`, but should that be streamlined to use the `v-container` approach instead of custom css? Maybe

Reply:
I agree. Maybe you can find the courage to make that change in MainContentArea? :) 

Then again this may be pushed as is but I'd prefer not using MainContentArea at all or using it every time it 'should' be used.

Reply:
Yeah I'll try to figure out something. If we do have a specific `MainContentArea` component, it should also be documented when it should be used.

Reply:
Renamed the `MainContentArea` to `SubViewContainer` so it is a little more descriptive now. It also uses the `v-container` component now.

Reply:
Yup, ready to merge